medusa/tv/series.py

```diff
@@ -548,6 +549,13 @@ def release_required_words(self):
         """Return release ignore words."""
         return [v for v in (self.rls_require_words or '').split(',') if v]
 
+    @staticmethod
+    def normalize_status(series_status):
+        """Return a normalized status given current indexer status."""
+        if not series_status:
+            return 'Unknown'
```

Comment:
Or we use this as default unknown status. Or we put it in the STATUS_MAP. So this could be a little bit more DRY.